src/Resources/ResourceManager/Properties/Resources.resx

```diff
@@ -423,4 +423,7 @@
   <data name="InvalidFilePath" xml:space="preserve">
     <value>The provided file {0} doesn't exist</value>
   </data>
+  <data name="NoSubscriptionsUnderCurrentDirectory" xml:space="preserve">
+    <value>No subscriptions can be found under current directory</value>
```

Comment:
We would like to give customer suggestions when this happen. How about this?
```suggestion
    <value>No subscriptions can be found under current directory. Please login again using Connect-AzAccount.</value>
```

Reply:
good idea